proposals/0269-implicit-self-explicit-capture.md

```diff
@@ -3,7 +3,7 @@
 * Proposal: [SE-0269](0269-implicit-self-explicit-capture.md)
 * Author: [Frederick Kellison-Linn](https://github.com/jumhyn)
 * Review Manager: [Ted Kremenek](https://github.com/tkremenek)
-* Status: **Accepted**
+* Status: **Implemented**
```

Comment:
SE-0269 is only implemented in the master branch, so we won't know which Swift version (e.g. 5.3 or 6) until probably next month.

(The previous announcements for [Swift 5.1][], [Swift 4.2][], and [Swift 4.0][] were in February.)

Until then, please could we leave this as accepted? Otherwise SE-0269 will disappear from the [status page][].

[Swift 5.1]: <https://swift.org/blog/5-1-release-process/>
[Swift 4.2]: <https://swift.org/blog/4-2-release-process/>
[Swift 4.0]: <https://swift.org/blog/swift-4-0-release-process/>
[status page]: <https://apple.github.io/swift-evolution/>